templates/vue/src/utils/Helpers.js

```diff
@@ -384,10 +384,21 @@ export default class Helpers {
       unlocked: true,
       accessible: true,
       reviewComments: [],
+      isDyad: false,
       popup: null,
     }
     return Helpers.deepMerge(baseNode, overrides)
   }
+
+  static nodeAndUserAreDyad(node) {
+    let roles = wp.getCurrentUser().roles
+    if (!node.isDyad) {
+      return false
+    } else {
+      return roles.includes("dyad")
+    }
```

Comment:
Could possibly reduce to
```suggestion
    return !node.isDyad ? false : roles.includes("dyad")
```